src/components/style.css

```diff
@@ -256,3 +256,7 @@ img.vue-lb-modal-image {
 .fade-enter, .fade-leave-to {
   opacity: 0;
 }
+
+.no-scroll {
+  overflow-y: hidden;
+}
```

Comment:
A little change: can you add a new line to the end of this file? Sorry I didn't notice it until now.